tests/scenario_tests/test_events_ignore_self.py

```diff
@@ -38,9 +39,9 @@ def handle_app_mention(say):
         response = app.dispatch(request)
         assert response.status == 200
         assert_auth_test_count(self, 1)
-        sleep(1)  # wait a bit after auto ack()
+        sleep(0.5)  # wait a bit after auto ack()
```

Comment:
Is this still necessary?

Reply:
Yess this is still required since we are asserting that an endpoint was not executed
If we validate without waiting we may assert the request was not sent before it gets sent